MovieApp/AppViewController.swift

```diff
@@ -311,22 +298,36 @@ class ExampleViewController: UIViewController {
         person_tag.trailingAnchor.constraint(equalTo: view1.trailingAnchor).isActive = true
         
         
+        
         let job_tag = UITextView()
         view1.addSubview(job_tag)
         job_tag.text = job
         job_tag.textColor = .black
         job_tag.isEditable = false
         job_tag.isScrollEnabled = false
         job_tag.font = UIFont.systemFont(ofSize: fontSizeMedium - 1)
+        
         job_tag.translatesAutoresizingMaskIntoConstraints = false
         job_tag.topAnchor.constraint(equalTo: person_tag.bottomAnchor).isActive = true
         job_tag.leadingAnchor.constraint(equalTo: view1.leadingAnchor).isActive = true
         job_tag.trailingAnchor.constraint(equalTo: view1.trailingAnchor).isActive = true
         job_tag.bottomAnchor.constraint(equalTo: view1.bottomAnchor).isActive = true
         
         return view1
-        
+    }
+    
+    
+    private func updateContentSizeScrollView() {
+        DispatchQueue.main.async {
+            self.scrollView.contentSize.height = self.mainView.frame.size.height
+        }
+    }
```

Comment:
Why do you need this. Having proper constraints in scrollView and in mainView would enable scrollView to automatically adjust its contentSize.